packages/components/src/components/index.tsx

```diff
@@ -18,6 +18,8 @@ const querySelector = (element: HTMLElement, selector: string) => {
 };
 
 export const CourierComponents: React.FunctionComponent = () => {
+  const preferencePageDraftMode =
+    window.courierConfig?.preferencePageDraftMode ?? false;
```

Comment:
we should also accept this attribute as prop in case a user wants to use this as pure react component.

Reply:
I have this supported for the PreferencePage component. The `CourierComponent` I thought was only for when you're using a script tag.